data/categories.txt

```diff
@@ -7957,6 +7957,48 @@ sw:Uwanja wa gofu|michezo
 fa:زمین گلف
 mr:गोल्फचे मैदान
 
+leisure-miniature_golf
+en:Mini-Golf|Minigolf|Miniature Golf|Putt Putt
```

Comment:
"Mini-Golf" can produce a mess in current categories matching implementation.
Remove this entry or combine it with leisure-golf_course.

Reply:
So the issue is in the hyphen like in https://github.com/organicmaps/organicmaps/pull/7774#discussion_r1550834728, right?

Here we can use a "Mini Golf" instead.

Reply:
No, Mini Golf - is the same. Category matching wirks for each token (now). Take it into account.
"Mini" will be confusing.
"Putt" also will match it.

Reply:
discussion: https://github.com/organicmaps/organicmaps/pull/7709#discussion_r1551967413

Reply:
So after the discussion my understanding is that while the "mini" part will certainly add some unrelated results (e.g. "mini-market") the full "mini golf" matches will be in the top most relevant results, so usability will be OK.

IMO it would be worse if typing "mini golf" (as its a very common term) will mix actual mini golf courses somewhere down inbetween unrelated stuff.

Reply:
At your decision, but IMO, ```en:4Minigolf|Miniature Golf|Putt Putt``` will be enough here with suggest.

Reply:
@vng I'm guessing there should be no spaces or dashes for every language, not just English, right?
Current translations (some have a dash, some have a space):


```
    en = Minigolf
    af = Minigolf
    ar = ميني غولف صغير
    az = Minigolf
    be = Міні-гольф
    bg = Миниголф
    ca = Minigolf
    cs = Minigolf
    da = Minigolf
    de = Minigolf
    el = Μίνι γκολφ
    es = Mini golf
    et = Minigolf
    eu = Minigolf
    fa = مینی گلف
    fi = Minigolf
    fr = Mini golf
    he = מיני גולף
    hi = मिनी गोल्फ
    hu = Minigolf
    id = golf mini
    it = Mini golf
    ja = ミニゴルフ
    ko = 미니골프
    lt = Minigolfas
    mr = मिनीगोल्फ
    nb = Minigolf
    nl = Minigolf
    pl = Mini golf
    pt = Minigolfe
    ro = Minigolf
    ru = Минигольф
    sk = Minigolf
    sv = Minigolf
    sw = Minigofu
    th = มินิกอล์ฟ
    tr = Mini golf
    uk = Міні-гольф
    vi = Golf mini
    zh-Hans = 迷你高尔夫
    zh-Hant = 迷你高爾夫
```

Reply:
Yes, take this and will merge:
```
    en = Minigolf
    af = Minigolf
    ar = ميني غولف صغير
    az = Minigolf
    be = Мінігольф
    bg = Миниголф
    ca = Minigolf
    cs = Minigolf
    da = Minigolf
    de = Minigolf
    el = Μίνι γκολφ
    es = Minigolf
    et = Minigolf
    eu = Minigolf
    fa = مینی گلف
    fi = Minigolf
    fr = Minigolf
    he = מיני גולף
    hi = मिनी गोल्फ
    hu = Minigolf
    it = Minigolf
    ja = ミニゴルフ
    ko = 미니골프
    lt = Minigolfas
    mr = मिनीगोल्फ
    nb = Minigolf
    nl = Minigolf
    pl = Minigolf
    pt = Minigolfe
    ro = Minigolf
    ru = Минигольф
    sk = Minigolf
    sv = Minigolf
    sw = Minigofu
    th = มินิกอล์ฟ
    tr = Mini golf
    uk = Мінігольф
    zh-Hans = 迷你高尔夫
    zh-Hant = 迷你高爾夫
```